docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -449,6 +447,9 @@ class DebugPage extends React.PureComponent<Props, State> {
   };
 
   private validateEndpointPath(endpointPath: string) {
+    if (!endpointPath) {
+      throw new Error('You must set the ENDPOINT PATH.');
```

Comment:
`specify the endpoint path`? (set -> specify, no reason to capitalize)

Reply:
Fixed, thanks!